stimela/backends/runner.py

```diff
@@ -52,6 +54,9 @@ def validate_backend_settings(backend_opts: Dict[str, Any], log: logging.Logger)
     selected = backend_opts.select or ["singularity", "native"]
     # select containerization engine, if any
     for name in selected:
+        # container tech cannot be used if cab.image has not been set
+        if name in ["singularity"] and not getattr(cab, "image", None):
+            continue
```

Comment:
If `selected` only includes `singularity`, but no cab was defined, will this loop behave as expected? Is there are reason for using `in` over `==`?